tests/rpmsigdig.at

```diff
@@ -2197,3 +2195,47 @@ runroot rpmkeys --define "_keyring openpgp" -Kv /data/RPMS/hello-2.0-1.x86_64-si
 ],
 [ignore])
 RPMTEST_CLEANUP
+
+RPMTEST_SETUP_RW([autosign setup])
+AT_KEYWORDS([autosign signature])
+
+rpmconf=$(rpm --eval "%{_rpmconfigdir}")
+
+RPMTEST_CHECK([
+runroot ${RPM_CONFIGDIR_PATH}/rpm-setup-autosign
+],
+[0],
+[],
+[Autosign already configured
+])
+
+RPMTEST_CHECK([
+runroot rm -rf /root/.config/rpm
+runroot ${RPM_CONFIGDIR_PATH}/rpm-setup-autosign -p gpg
+],
+[0],
+[],
+[stderr])
+
+RPMTEST_CHECK([
+cmd=$(grep ^sudo stderr | cut -c6-)
+test -n "${cmd}" && runroot ${cmd}
```

Comment:
This is why the test case fails: The `test` is unsuccessful (i.e. no "sudo ..." line in the output) so it returns non-zero.

Seems like the test just doesn't account for the message being absent. I guess it should just run the actual command without expecting it (since it won't be printed when non-interactive now).

Reply:
Righty, totally forgot this construct is there. It comes up in the GH action log as the culprit but still didn't connect...

I guess the idea was to ensure the command we output really works. So I think I'll just drop the "is a tty" test from the script for now, that's cosmetic at best anyway.

Reply:
I tried adding -v(erbose) and have it output the import command with that set, but somehow it didn't seem right at all. So I just discarded the logic for now, ie back to the original unconditional output. Having the sudo command output'ed in a scripted logic doesn't hurt anything afterall.

Reply:
Ack, the tty test is indeed cosmetic and not worth all this trouble :sweat_smile:

Reply:
> I guess the idea was to ensure the command we output really works.

Oh right, of course. Didn't realize...